tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+	"compilerOptions": {
+		"module": "nodenext",
+		"moduleResolution": "nodenext",
+		"moduleDetection": "force",
```

Comment:
The default value of `moduleDectection` is `auto`, which uses the `package.json` `type` field when `module` is `nodenext`. Since we use `type: 'module'` in `package.json`, it seems like we could remove the `moduleDetection: "force"` line?